### DISCUSSION TITLE
编辑器问题

### DISCUSSION THREAD

**Discussion by chengse66** [galacean/discussion#67] [Editor]
1.阴影断面
2.金属度设置为0后,接受不到阴影
3.第一次穿件sphere后,第二次创建sphere会丢失组件
4.场景的图标无法点击选中
5.目录树里的元素没法复制.
6.阴影缺少半影
其他还没测试,先这样吧.

**Comment by chengse66** [comment]
创建,错别字了~~

**Reply by yangfengzzz** [reply]
半影是什么意思？

**Comment by gz65555** [comment]
- 第 3 个问题我们看一下
- 第 4 个问题需要引擎 1.0 支持，应该需要到下个里程碑
- 第 5 个问题已经有了，这周会更新

**Comment by gz65555** [comment]
已经修复
